tests/core_tests/startup_tests.py

```diff
@@ -0,0 +1,37 @@
+
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2020 - 2022 Pionix GmbH and Contributors to EVerest
+
+import logging
+import pytest
+
+from everest.testing.core_utils.fixtures import *
```

Comment:
`import *` is really deprecated, are there any fixtures used in this test at all?